File: assets/AssignmentTwo/JS/SeriesTwo.js
let n = +prompt("How many number you enter: ", "");

let series = '';
for (let i = 2; i <= n; i = i + 2) {
  series += (1 / (i * i)).toFixed(2) + ' ';
}

console.log("\n\nSeries of : (1/1 + ½ + ⅓ +...................+1/n) is");
console.log("------------------------\n");
console.log(series);

File: assets/AssignmentTwo/C/SeriesTwo.c
#include <stdio.h>

int main(void) {
  int i, n;

  printf("How many number you enter: ");
  scanf("%d", &n);

  printf("\n\nSeries of : (½^2 +¼^2 +.......................+1/n^2) is \n");
  printf("------------------------\n\n");
  for (i = 2; i <= n; i= i + 2) {
    printf("%.2lf ", (double) 1 / (i * i));
  }
}